bans-core/src/main/java/space/arim/libertybans/core/config/Formatter.java

```diff
@@ -172,40 +176,50 @@ String getVariable() {
 
 		final long timePassed = now - start;
 
-		final String durationFormatted;
-		final String relativeEndFormatted;
+		final String durationFormatted, durationFormattedSimple;
```

Comment:
Is there a reason you decided to put the `durationFormattedSimple` variable here? As far as I can see, it's only used in one place. You don't even need the variable at all.